### PR TITLE
website: update community people section

### PR DESCRIPTION
**Pull request by phinze**
As the page says:

> Over time, faces may appear from this list as contributors come and
> go.

Let's reflect the list of peoople working on Terraform nowadays! :)

### CODE REVIEW

**Comment by phinze** [comment]
community_-_terraform_by_hashicorp

**Comment by phinze** [comment]
cc @catsby @jen20 feel free to suggest tweaks to language here. for now imma :shipit:

File: website/source/community.html.erb [hunk 1]
```diff
@@ -54,36 +54,40 @@ disappear from this list as contributors come and go.
...
-			Jack Pearkes is a creator of Terraform. He created and maintains
-			most of the providers and documentation.
-			He is also a core committer to
-			<a href="https://www.packer.io">Packer</a> and
-			<a href="https://www.consul.io">Consul</a>
-			while also being an employee of
-			<a href="https://www.hashicorp.com">HashiCorp</a>.
+      Clint Shryock is a HashiCorp Engineer working on Terraform. He is the
+      primary maintainer of the AWS provider, and works across all providers.
+      Client was also the primary author of the Fastly provider.
```
Comment (catsby) [inline website/source/community.html.erb]: > Client was also

Curse you autocorrect! ౿(ఠ_ఠఎ)

Comment (phinze) [inline website/source/community.html.erb]: HA thanks for looking. Fixing now

**Comment by jen20** [comment]
LGTM.

**Pull request merged**